### PR TITLE
No shapes in profilings by default

### PR DESCRIPTION
**Pull request by astachowiczhabana**
Disable shapes in profiling by default

### CODE REVIEW

File: examples/text-generation/run_generation.py
Comment (pramodkumarjuit) [inline examples/text-generation/run_generation.py:1]: Shouldn't we add 'profiling_record_shapes=args.profiling_record_shapes'  here as well?

Comment (pramodkumar-habanalabs) [inline examples/text-generation/run_generation.py:1]: 1. Shouldn't we add 'profiling_record_shapes=args.profiling_record_shapes' here as well?
2. Can we make 'with_stack' an argument based, keeping the default value as False?

Comment (astachowiczhabana) [reply examples/text-generation/run_generation.py:1]: 1. Where? You comment refers to the whole file, can you please specify exact place
2. Done

Comment (pramodkumar-habanalabs) [reply examples/text-generation/run_generation.py:1]: 1. New line no. 377.

**Review by pramodkumar-habanalabs — APPROVED** [review]
Looks good.

**Comment by astachowiczhabana** [comment]
https://github.com/huggingface/optimum-habana/pull/1050/

**Pull request merged**